web-animations-1/Overview.bs

```diff
@@ -796,9 +831,22 @@ An animation's start time is initially <a>unresolved</a>.
 
 An <a>animation</a> also maintains a <dfn>hold time</dfn> <a>time value</a>
 which is used to fix the animation's output <a>time value</a>, called its
-<a>current time</a>, in circumstances such as pausing</a>.
+<a>current time</a>, in circumstances such as pausing.
 The <a>hold time</a> is initially <a>unresolved</a>.
 
+An <a>animation</a> also maintains a <dfn>hold phase</dfn>
+[=timeline phase|phase=] which is used by <a>animation effect</a> to calculate
```

Comment:
This currently reads:

"An animation also maintains a hold phase phase" (← notice the double "phase").

Reply:
Removing holde phase concept for now. Going to add this concept in a later PR that covers the effects in more detail.